.idea/codeStyles/Project.xml

```diff
@@ -23,6 +23,22 @@
       </option>
     </JavaCodeStyleSettings>
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```

Comment:
I know this is a draft but this edits should be reverted since they are not mandatory for the related issue. We change only relevant lines :)